packages/foam-vscode/src/utils/grouped-resources-tree-data-provider.ts

```diff
@@ -248,7 +249,6 @@ class ResourceTreeItem extends vscode.TreeItem {
         arguments: [resource.uri],
       };
     }
-    this.contextValue = itemLabel;
```

Comment:
I think this got here by mistake so I removed it - pinging @leonhfr and @joeltjames for confirmation that we don't need it

Reply:
confirmed it's not needed